projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.scss

```diff
@@ -0,0 +1,44 @@
+.jetpack-email-preview {
+  height: 200px;
+  width: 645px;
+  left: 391px;
+  top: 392px;
+  border-radius: 0;
+  .components-modal__header-heading {
+    margin-top: 30px;
+    font-size: 26px;
+  }
+  .components-modal__header {
+    margin-left: 17px;
+  }
+  .components-modal__content {
+    margin-left: 17px;
+  }
+}
+
+.jetpack-email-preview__email {
+  .components-text-control__input {
+    height: 44px;
+    width: 266px;
+    border-radius: 3px;
+    border-color: gray;
+    padding: 12px 16px 12px 16px;
+    background-color: lightgray;
+  }
+}
+
+.jetpack-email-preview__button {
+  border-radius: 2px;
+  padding: 16px 24px 16px 24px;
+  width: 80px;
+  height: 44px;
+}
+
+.jetpack-email-preview__img {
+  height: 110px;
+}
+
+.jetpack-email-preview__email-sent {
+  color: green;
```

Comment:
Let's use colour variable.

Would one from core's `@wordpress/base-styles` be close to designed? https://github.com/WordPress/gutenberg/blob/053413e9e7e1053037432a57931e53d7d03c7060/packages/base-styles/_colors.scss#L28

I think there was Jetpack base-styles package as well to pick some colours from.

Reply:
> I think there was Jetpack base-styles package as well to pick some colours from.

Yes, that's what I would recommend as well. The `$studio-jetpack-green-` colors could be useful here. Here is an example of how you can use it:
https://github.com/Automattic/jetpack/blob/3a3c4adc1c0913bac224ddb832713bd6101fe1ab/projects/plugins/jetpack/extensions/plugins/publicize/editor.scss#L2-L15

You can read more about it here:
https://github.com/Automattic/jetpack/blob/ade2b4cad370b6feb6153f90f7f1cf8e9dfc9d77/projects/plugins/jetpack/extensions/README.md#colors

Reply:
Thanks!